cacheops/invalidation.py

```diff
@@ -18,16 +18,28 @@
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+def delete_invalid_caches(conj_keys):
+    for conj_key in conj_keys:
+        cache_keys = redis_client.sscan_iter(conj_key, count=1000)
+        for batch in cache_keys:
+            redis_client.srem(conj_keys, batch)
+            redis_client.delete(batch)
+    redis_client.delete(conj_keys)
```

Comment:
This moves deletion of invalid cache keys from Lua into Python, where it is no longer atomic – but we no longer care, since the keys have already been renamed. It's also much slower – but it'll actually be done asynchronously, without holding up Rover.